src/consensus/consensus.h

```diff
@@ -12,15 +12,16 @@
 
 /** The maximum allowed size for a serialized block, in bytes (only for buffer size limits) */
 static const unsigned int MAX_BLOCK_SERIALIZED_SIZE = 4000000;
-/** The maximum allowed weight for a block, see BIP 141 (network rule) */
-static const unsigned int MAX_BLOCK_WEIGHT = 4000000;
-/** The maximum allowed number of signature check operations in a block (network rule) */
-static const int64_t MAX_BLOCK_SIGOPS_COST = 80000;
+/**
+  * The maximum allowed size for a block, in bytes (network rule).
+  */
+static const unsigned int MAX_BLOCK_BASE_SIZE = 1000000;
+/** 
+ * The maximum allowed number of signature check operations in a block (network rule) */
+static const int64_t MAX_BLOCK_SIGOPS_COST = 20000;
```

Comment:
if this needs to be relative to block size, it can't be a constant. we need to add it to federation parameters and change it when block size changes. In this PR I'll just add the above line. In #234 I'll make the federation param changes.

Reply:
Federation param (or XField history) change is not in PR234. It will be a new PR.